llvm/lib/Transforms/Vectorize/VPlanValue.h

```diff
@@ -202,7 +202,6 @@ class VPUser {
   /// Subclass identifier (for isa/dyn_cast).
   enum class VPUserID {
     Recipe,
-    LiveOut,
   };
```

Comment:
Perhaps better clean this up for now, to avoid false pretense it is in use, and reintroduce if/when desired (for VF/TC).